src/engine/Actions/Action.ts

```diff
@@ -373,7 +373,7 @@ export class RotateTo implements IAction {
       this._rotationType = rotationType || RotationType.ShortestPath;
    }
 
-   public update(delta: number): void {
```

Comment:
@kamranayub Are these just unused arguments? I assume they are since nothing broke, and the new compiler config probably complains about this.